include/TMU.hpp

```diff
@@ -33,21 +37,21 @@ class TMU {
      */
     uint16_t getObjectDictionarySize() const;
 
+    /**
+     * Updates the thermTemps values
+     */
+    void updateTemps();
+
 private:
     /**
      * Stores the 4 16-bit temperature values.
      */
     uint16_t thermTemps[4] = {};
 
-    SPI& spi;
-    DEV::MAX31855 MAXs[4] = {
-        DEV::MAX31855(spi, 0),
-        DEV::MAX31855(spi, 1),
-        DEV::MAX31855(spi, 2),
-        DEV::MAX31855(spi, 3)
-    };
-
-    void updateTemps();
+    /**
+     * Stores the 4 MAX31855 thermocouples
+     */
+    DEV::MAX31855 thermocouples[4];
```

Comment:
Is this proper grammar? Shouldn't it be named thermocouple[4] as _thermocouples_ implies multiple _thermocouples_ per index?

Reply:
No, `thermocouples` is a variable referring to an array of multiple thermocouple objects, so the grammar is perfectly fine. If we wanted to make it more explicit, we could rename it `thermocoupleArray`, in which case the word thermocouple is acting as an adjective, so it obviously would not be made plural.

Reply:
Thank you for the clarification.